src/zen/forks/fork.cpp

```
@@ -76,16 +80,20 @@ void Fork::setCommunityFundAddressMap(const std::map<CBaseChainParams::Network,s
  * @return the community fund addresses for this fork and network
  */
 const std::vector<std::string>& Fork::getCommunityFundAddresses(CBaseChainParams::Network network, CommunityFundType cfType) const {
-    if (cfType == CommunityFundType::FOUNDATION) {
+    switch (cfType) {
+    case CommunityFundType::FOUNDATION:
         return communityFundAddressMap.at(network);
-    }
-    if (cfType == CommunityFundType::SECURENODE) {
+        break;
+    case CommunityFundType::SECURENODE:
         return secureNodeFundAddressMap.at(network);
-    }
-    if (cfType == CommunityFundType::SUPERNODE) {
+        break;
+    case CommunityFundType::SUPERNODE:
         return superNodeFundAddressMap.at(network);
+        break;
+    default:
+        return communityFundAddressMap.at(network);
```

this should throw an exception